src/test/java/com/commercetools/sync/products/utils/productvariantupdateactionutils/prices/BuildProductVariantPricesUpdateActionsTest.java

```diff
@@ -21,8 +21,53 @@
 import java.util.List;
 
 import static com.commercetools.sync.products.utils.ProductVariantUpdateActionUtils.buildProductVariantPricesUpdateActions;
-import static com.commercetools.sync.products.utils.productvariantupdateactionutils.prices.PriceDraftFixtures.*;
-import static com.commercetools.sync.products.utils.productvariantupdateactionutils.prices.PriceFixtures.*;
+import static com.commercetools.sync.products.utils.productvariantupdateactionutils.prices.PriceDraftFixtures.DRAFT_DE_100_EUR_01_02_CHANNEL1_CUSTOMTYPE1_CUSTOMFIELDX;
```

Comment:
Can we please leave these imports to be imported with a wild card? It was done on purpose here due to the huge amount of constants from the same class.

Reply:
😅  IntelliJ Idea changed it probably, Ok, I will revert.

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/320/commits/395547b2d36451f6ce8a2cbfee3070f440bfba5c